primer/brand
# OrderedList and UnorderedList Components

![image](https://user-images.githubusercontent.com/31220082/207701512-85e3d5b9-a2fb-438e-8ced-6e35d1e306b4.png)


## Summary

Two list component 'unordered' and 'ordered' 

## List of notable changes:

- **added** UnorderedList component
- **added** OrderedList component

## What should reviewers focus on?

- Making sure the components work as expected

## Steps to test:

1. Open the storybook instance in CI-deployed preview environment
2. Go to both UnorderedList and OrderedList stories in Storybook
3. Verify that both components behaves as described.

## Supporting resources (related issues, external links, etc):

- [Design Ticket](https://github.com/github/primer/issues/1587)
- [Component Epic](https://github.com/github/primer/issues/1352)

## Contributor checklist:


## Reviewer checklist:


## Preview Deployments:
- [Markdown Docs](https://primer-1e22b62640-26139705.drafts.github.io/)
- [Storybook](https://primer-1e22b62640-26139705.drafts.github.io/storybook)
